src/capitalsFirst.js

```diff
@@ -17,8 +17,28 @@
  *
  * @returns {string} - string with uppercase words in front
  */
+function isUppercaseLetter(str) {
+  return (str >= 'A' && str <= 'Z');
+}
+
+function isLowercaseLetter(str) {
+  return (str >= 'a' && str <= 'z');
+}
+
 function capitalsFirst(str) {
-  // write code here
+  const initialArray = str.split(' ');
+  const uppercaseLetters = [];
+  const lowercaseLetters = [];
+
+  initialArray.forEach(function(i) {
```

Comment:
use [arrow functions](https://mate-academy.github.io/style-guides/javascript.html#arrows--use-them) in callbacks